docs/development-guidelines/operational-guidelines.md

```diff
@@ -0,0 +1,99 @@
+# Operational guidelines
+
+The following guidelines are meant to help teams and contributors organize themselves and establish roles within template authoring flows to ensure consistency, quality, and collaboration.
+
+## Template owner(s)
+
+The template owner is the person who owns the template and acts as a point of contact to receive notification when applicable. Typically this role is filled by a core maintainer, who may or may not be code owner, but probably has some administrative persmissions in the overall repository.
+
+This code owner may the one organizing the work and reporting progress. 
+
+## Project (technical lead)
+
+A project technical lead may be the template owner, or a designated team member representing the technology, tool or service to high-light with the purpose of providing technical direction, coherence, and quality across the project. 
+
+A technical lead, depending on their seniority and competence, may be the ultimate decision maker and coordinator of all technical efforts, although when templates integrate multiple tools and service, we recommend to scope per technology and have experts on each category assuming this role.
+
+## Case scenarios and specification
+
+Before the team starts coding, make sure to have designed scenarios, use-cases and specification, answering thes questions
+
+- What services, tools and products are we showcasing?
+- What are our technical requirements?
+- What is the use case/functional specification? 
+    * Who will interact with your application?
+    * How does the user interact with the application?
+    * What does the application take as input?
+    * What is the application output?
+- What are the non-functional specifications, apart from the perfomance, security and usability success criteria defined by the standardization effort?
+
+### Creating a backlog
+
+Once you have the use-case and specification in place, create a backlog for all features to be developed, and prioritize them. Assign issues to respective experts, and check the progress on a regular basis.
+Use the backlog to create scopes mapping to [milestones](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/about-milestones), and assign them to the corresponding milestone.
+
+Progress can be shared during standups or async on a Teams chat.
+
+When the issue is completed, close it, referencing the pull request that has addressed it. For example comment and close with `Fixed by #[number of PR]`
```

Comment:
Recommendation to add a reminder that it's a good practice to close with comment for folks to follow along with the context async.